### PR TITLE
DEV: Switch to Chromium in CI

### PR DESCRIPTION
**Pull request by Flink**
This PR switches to using Chromium in our CI jobs instead of Chrome.

### CODE REVIEW

File: app/assets/javascripts/discourse/testem.js [hunk 1]
```diff
@@ -127,14 +127,13 @@ class Reporter extends TapReporter {
 module.exports = {
   test_page: "tests/index.html?hidepassed",
   disable_watching: true,
-  launch_in_ci: ["Chrome"],
-  // launch_in_dev: ["Chrome"] // Ember-CLI always launches testem in 'CI' mode
+  launch_in_ci: ["Chromium"],
```
Comment (davidtaylorhq) [inline app/assets/javascripts/discourse/testem.js:130]: IIRC this config option is badly named, and ember-cli runs testem in 'CI' mode, even in local development.

Comment (Flink) [reply app/assets/javascripts/discourse/testem.js:130]: Their docs say `launch_in_ci:  [Array]  list of launchers to use for CI runs`, so I guess this is a list of launchers to use if none is specified maybe? I’m not familiar at all with testem :sweat_smile:

**Review by davidtaylorhq — APPROVED** [review]

File: app/assets/javascripts/discourse/testem.js [hunk 2]
```diff
@@ -127,14 +127,25 @@ class Reporter extends TapReporter {
...
+      "--disable-software-rasterizer",
+      "--mute-audio",
+      "--remote-debugging-port=4201",
+      "--window-size=1440,900",
+      "--enable-precise-memory-info",
+      "--js-flags=--max_old_space_size=4096",
+    ].filter(Boolean),
     Chrome: [
-      // --no-sandbox is needed when running Chrome inside a container
+      // --no-sandbox is needed when running Chromium inside a container
```
Comment (CvX) [inline app/assets/javascripts/discourse/testem.js:148]: aren't those still Chrome flags? 🙂 

```suggestion
      // --no-sandbox is needed when running Chrome inside a container
```

Comment (Flink) [reply app/assets/javascripts/discourse/testem.js]: Ah just had the same idea :sweat_smile: :+1:

**Pull request merged**